app/src/main/java/com/mealsmadeeasy/ui/home/glance/WeekAtAGlanceMealViewHolder.kt

```diff
@@ -1,19 +1,42 @@
 package com.mealsmadeeasy.ui.home.glance
 
+import android.support.v7.app.AlertDialog
+import android.support.v7.widget.AppCompatImageView
 import android.support.v7.widget.RecyclerView
+import android.view.Gravity
 import android.view.View
 import android.widget.ImageView
+import android.widget.PopupMenu
```

Comment:
We need to use the support version of this class, so import `android.support.v7.widget.PopupMenu` instead. For future reference, if there's a support or appcompat class for something, use it (this doesn't apply to `TextView` or `ImageView` however).